wgpu-core/src/command/render.rs

```diff
@@ -1619,9 +1627,18 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         indexed,
                     } => {
                         let scope = if indexed {
-                            PassErrorScope::DrawIndexedIndirect
+                            PassErrorScope::Draw {
+                                indexed: true,
```

Comment:
now we can merge those two branches into one:
```rust
let scope = PassErrorScope::Draw {
  indexed,
  indirect: true,
  pipeline: state.pipeline.last_state,
};
```

Reply:
oh duh, of course :)